### PR TITLE
fix(ui): exclude notifications without the status

### PR DESCRIPTION
**Pull request by userquin**
This PR fixes the error when accessing the tab mentions in notifications with ElkDev, there are a few mention notifications without status:

imagen

### CODE REVIEW

File: components/notification/NotificationPaginator.vue [hunk 1]
```diff
@@ -13,6 +13,14 @@ const virtualScroller = false // TODO: fix flickering issue with virtual scroll
 
 const groupCapacity = Number.MAX_VALUE // No limit
 
+function includeNotificationsForStatusCard({ type, status }: mastodon.v1.Notification) {
+  // Exclude update, mention, pool and status notifications without the status entry:
+  // no makes sense to include them
+  // Those notifications will be shown using StatusCard SFC:
+  // check NotificationCard SFC L68 and L81 => :status="notification.status!"
+  return !((type === 'update' || type === 'mention' || type === 'poll' || type === 'status') && !status)
```
Comment (ayoayco) [inline components/notification/NotificationPaginator.vue:21]: maybe more clear if

```suggestion
  return status && !['update', 'mention', 'poll', 'status'].includes(type)
```

Comment (userquin) [reply components/notification/NotificationPaginator.vue:21]: Status is not required in some types

Comment (userquin) [reply components/notification/NotificationPaginator.vue:21]: @ayoayco you suggestion is wrong, should be `return status || ![...`

Comment (userquin) [reply components/notification/NotificationPaginator.vue]: done

File: components/notification/NotificationPaginator.vue [hunk 2]
```diff
@@ -13,6 +13,14 @@ const virtualScroller = false // TODO: fix flickering issue with virtual scroll
 
 const groupCapacity = Number.MAX_VALUE // No limit
 
+function includeNotificationsForStatusCard({ type, status }: mastodon.v1.Notification) {
+  // Exclude update, mention, pool and status notifications without the status entry:
+  // no makes sense to include them
+  // Those notifications will be shown using StatusCard SFC:
+  // check NotificationCard SFC L68 and L81 => :status="notification.status!"
+  return status || !['update', 'mention', 'poll', 'status'].includes(type)
```
Comment (patak-dev) [inline components/notification/NotificationPaginator.vue:21]: I think it would be good to extract this array out from the function

Comment (userquin) [reply components/notification/NotificationPaginator.vue]: done

**Review by ayoayco — APPROVED** [review]
👌

File: components/notification/NotificationPaginator.vue (hunk 2 above)
Comment (userquin) [reply components/notification/NotificationPaginator.vue]: @patak-dev don't forget this one :pray:

**Review by patak-dev — APPROVED** [review]

**Pull request merged**